controllers/util/testUtil.go

```diff
@@ -30,6 +32,25 @@ func New(ctx context.Context, k8sClient client.Client, r reconcile.Reconciler) *
 	}
 }
 
+// RandCustomer Generates random customer names and emails
+func (t *TestUtil) RandCustomer(customer jiraservicedeskv1alpha1.Customer) jiraservicedeskv1alpha1.Customer {
```

Comment:
Comment should be something like RandCustomer updates a customer with a random name and email

Reply:
ack